### PR TITLE
Containerized

### PR DESCRIPTION
**Pull request by T4ylan**
- added Containerfile
- ability to customize the proxies (nexus) via ENV CUSTOM_REPOSITORIES

### CODE REVIEW

File: Containerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+COPY gradlew .
+COPY build.gradle .
+COPY src src
+COPY gradle.properties .
+
+# Build application and test it
+RUN ./gradlew assemble --no-daemon && \
+	./gradlew testClasses --no-daemon
+
+FROM quay.io/odpi/egeria:latest
```
Comment (T4ylan) [inline Containerfile]: TODO: parametrize the egeria and version

**Review by juergenhemelt — COMMENTED** [review]
I would omit the gradle build in the Dockerfile because that is done in Github Actions.

**Comment by planetf1** [comment]
> I would omit the gradle build in the Dockerfile because that is done in Github Actions.

It is for various other repositories, however the build changes have not been made to this sample yet. 
They use a standard dockerfile 'Dockerfile' and build an image by taking the core egeria image as base, and then adding in just this single connector, using the existing build. They are run in the action after the build has been executed.

I think it's entirely reasonable to have additional Dockerfile for other purposes - including for example one that builds from source, and this wouldn't get used by the opensource ci/cd process. But you need to figure out the right base - for egeria we tend to use the redhat UBI images (secure), and ensure the egeria chassis itself is included at the current levels. This is another thing built into the dockerfile parameterization in our other repos (like for strimzi) - it matches the egeria base image with that expressed by dependency.

**Comment by planetf1** [comment]
I can update the build scripts for this repo to align with what we've done elsewhere like for strimzi if it helps.

**Review by juergenhemelt — APPROVED** [review]

**Review by juergenhemelt — APPROVED** [review]

**Comment by T4ylan** [comment]
I'd close this PR, as there is a take2 here: https://github.com/odpi/egeria-connector-integration-lineage-event-driven-sample/pull/52

**Pull request closed without merging**